plugins/woocommerce/templates/cart/mini-cart.php

```diff
@@ -44,7 +44,7 @@
 						sprintf(
 							'<a href="%s" class="remove remove_from_cart_button" aria-label="%s" data-product_id="%s" data-cart_item_key="%s" data-product_sku="%s">&times;</a>',
 							esc_url( wc_get_cart_remove_url( $cart_item_key ) ),
-							esc_attr__( 'Remove this item', 'woocommerce' ),
+							sprintf( esc_attr__( 'Remove %s', 'woocommerce' ), $product_name ),
```

Comment:
Same as the suggestion above, we should do `esc_attr` on the final result.

```suggestion
							esc_attr( sprintf( __( 'Remove %s', 'woocommerce' ), $product_name ) ),

```

We also need a `/* translators:` comment above this use of `__`